### DISCUSSION TITLE
Github Copilot at work

### DISCUSSION THREAD

**Discussion by vladimir-sf** [community/community#38458] [Copilot Conversations] [answered]
I have a couple of questions about the GitHub copilot.
1) I want to use GitHub copilot at work, I have a personal license, but I don't know if GitHub allows the use of an individual license at work. Can someone point me in the right direction?

2) If I uncheck "Allow GitHub to use my code snippets for product improvements," does it mean that GitHub copilot will not send my code to the servers and will process everything on my machine?

**Comment by davecheney** [comment, marked as answer]
1. That is a discussion between yourself and your employer.
2. Copilot must send code snippets to the our servers to provide context for a completion, however we will not record what was sent or our reply if you do not give permission to do so.

**Reply by cmp-nct** [reply]
1) he asked if Github allows it, his employer won't have an answer on that ..

**Comment by idoraquel** [comment]
@davecheney could you please point out where this permission is set (using individual version)?
>however we will not record what was sent or our reply if you do not give permission to do so

**Reply by davecheney** [reply]
github.com/settings/copilot
